includes/wf_crm_admin_form.inc

```diff
@@ -1738,7 +1738,7 @@ class wf_crm_admin_form {
             }
           }
           elseif ($field['type'] === 'hidden' && !empty($field['expose_list'])) {
-            $elements = $this->webform->getElementsDecoded();
+            $elements = $this->webform->getElementsDecodedAndFlattened();
```

Comment:
👍 This change works for me! 

The basics of it are, from within the Webform class

```
// initElements()
$elements = Yaml::decode($this->elements);
$this->elementsDecoded = $elements; // returned by getElementsDecoded, full nested YAML basically

// initElementsRecursive
$this->elementsDecodedAndFlattened[$key] = WebformElementHelper::getProperties($element);
```

So flattened makes it easier to work with and we don't need to worry about nesting. This would probably make things easier in a few areas as well.